packager/hls/base/master_playlist.cc

```diff
@@ -364,11 +372,12 @@ void BuildMediaTag(const MediaPlaylist& playlist,
   out->append("\n");
 }
 
-void BuildMediaTags(
-    std::list<std::pair<std::string, std::list<const MediaPlaylist*>>>& groups,
+std::list<MediaTagslist> BuildMediaTags(
```

Comment:
This seems a bit confusing, before this change `BuildMediaTags` called `BuildMediaTag` within, but now it no longer does that. We probably need to rename this function.

Reply:
Or alternatively keep this function as is, giving it a list of of `MediaTagslist` or whatever we rename that to, and instead build the right struct (with the group name, etc) in the main loop as suggested in another comment